tools/tests/dropdown/attributes.test.js

```diff
@@ -0,0 +1,109 @@
+
+const attributesTemplate = `<gameface-dropdown class="gameface-dropdown-component">
+<dropdown-option slot="option">Cat</dropdown-option>
+<dropdown-option slot="option">Dog</dropdown-option>
+<dropdown-option slot="option">Giraffe</dropdown-option>
+<dropdown-option slot="option">Lion</dropdown-option>
+<dropdown-option slot="option" disabled>Pig</dropdown-option>
+<dropdown-option slot="option">Eagle</dropdown-option>
+<dropdown-option slot="option">Parrot</dropdown-option>
+<dropdown-option slot="option">Last Parrot</dropdown-option>
+</gameface-dropdown>`;
+
+const dropdownSelector = '.dropdown-attributes-test-wrapper gameface-dropdown';
+
+/**
+ * Method initializing dropdown test page
+ * @param {string} templateString
+ * @returns {Promise<void>}
+ */
+function setupDropdownAttributesTestPage(templateString) {
+    const el = document.createElement('div');
+    el.className = 'dropdown-attributes-test-wrapper';
+    el.innerHTML = templateString;
+
+    cleanTestPage('.dropdown-attributes-test-wrapper');
+
+    document.body.appendChild(el);
+
+    return new Promise((resolve) => {
+        waitForStyles(resolve);
+    });
+}
+
+// eslint-disable-next-line max-lines-per-function
+describe('Dropdown Attributes', () => {
+    afterAll(() => {
+        document.querySelector('.dropdown-attributes-test-wrapper').querySelector('.guic-dropdown-options').innerHTML = '';

```

Comment:
Why do we clean the options here :D? Isn't the `cleanTestPage` removes the whole dropdown from the page as well as its options?

Reply:
Not sure, copy-pasted from another test I guess.